src/WebJobs.Script.Grpc/Proto/FunctionRpc.proto

```diff
@@ -252,4 +252,5 @@ message RpcHttp {
   string status_code = 12;
   map<string,string> query = 15;
   bool is_raw = 16;
+  TypedData rawBody = 17;
```

Comment:
I think that this will always be a `string`, so you can concretely type it as a string.

Reply:
Unfortunately, it's not always a `string`, sometimes it's `None`. I could have created a new type which only has those types, but I opted to reuse the existing TypedData object since it wasn't very opinionated.

Reply:
Correct - in those cases, would not setting `rawBody` be sufficient? Protobuf doesn't require all properties to be set in a message, so I think that all client libs will handle the None case. I'd suggest that this change is scoped only to node, which definitely handles nullables. To enable this behavior for node & other workers in the future we could pass the body bytes directly to the client, where they can determine rawbody

That said, I'm on vacation and `TypedData` works. Just my 2¢ :)

Reply:
If I don't set it, it's `null`. `null` !== `undefined` which is a breaking behavior. protobuf for Node.js doesn't seem to have an option for "null" - looks like a fun thread: https://github.com/google/protobuf/issues/1606